Sources/SceytChatUIKit/Providers/ChannelMessageMarkerProvider.swift

```diff
@@ -113,18 +113,70 @@ open class ChannelMessageMarkerProvider: Provider {
             return
         }
         logger.debug("[MARKER CK] send \(ids.count)")
-        switch markerName {
-        case DefaultMarker.received:
+        switch DefaultMarker(rawValue: markerName) {
+        case .received:
             channelOperator.markMessagesAsReceived(ids: ids.map { NSNumber(value: $0)}, completion: completed(_:error:))
-        case DefaultMarker.displayed:
+        case .displayed:
             channelOperator.markMessagesAsDisplayed(ids: ids.map { NSNumber(value: $0)}, completion: completed(_:error:))
         default:
             channelOperator.markMessages(markerName: markerName, ids: ids.map { NSNumber(value: $0)}, completion: completed(_:error:))
         }
     }
 }
 
-enum DefaultMarker {
-    static let displayed = "displayed"
-    static let received = "received"
+extension ChannelMessageMarkerProvider {
+    
+    open func loadMarkers(_ query: MessageMarkerListQuery, completion: ((Error?) -> Void)? = nil) {
+        guard chatClient.connectionState == .connected else { return }
```

Comment:
@arthursct  completion block always must be called 
if not connected then completion?(SceytChatError.notConnect)
if loading then SceytChatError.queryInProgress

or just call completion?(nil) if no error and no success